apps/system/watchtower.yml

```diff
@@ -0,0 +1,32 @@
+---
+version: '3'
+services:
+  watchtower:
+    hostname: 'watchtower'
+    container_name: 'watchtower'
+    image: 'ghcr.io/containrrr/watchtower:latest'
+    environment:
+      - 'PUID=${ID}
+      - 'PGID=${ID}'
+      - 'TZ=${TZ}'
+      - 'WATCHTOWER_CLEANUP=true'

```

Comment:
Environment  !!!!

Reply:
For Cleanup?

Reply:
It's better for the hole part 

The goal is ..  all can be changed  <| the goal of DS |>